### PR TITLE
Add gitlab-ci-multi-runner v0.6.2 (new formula)

### PR DESCRIPTION
**Pull request by cybertk**
New formula for [gitlab-ci-multi-runner](https://gitlab.com/gitlab-org/gitlab-ci-multi-runne)

### CODE REVIEW

File: Library/Formula/gitlab-runner.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      commit_sha = `git rev-parse --short HEAD`
+
+      # Copy from Makefile
+      system "go", "build", "-o", "gitlab-runner", "-ldflags", "-X main.NAME=gitlab-runner -X main.VERSION=#{version} -X main.REVISION=#{commit_sha}"
+      bin.install "gitlab-runner"
+    end
+  end
+
+  test do
+    assert_match /gitlab-runner version #{version} (.*)/, shell_output("gitlab-runner --version")
```
Comment (bfontaine) [inline Library/Formula/gitlab-runner.rb]: You can remove this ` (.*)`.

File: Library/Formula/gitlab-runner.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class GitlabRunner < Formula
+  desc "The official GitLab CI runner written in Go"
+  homepage "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner"
+  url "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner.git", :tag => "v0.6.2", :revision => "3227f0aa5be1d64d2ec694bd3758e0d43e92b36b"
```
Comment (DomT4) [inline Library/Formula/gitlab-runner.rb]: Please break the `tag` and `revision` onto their own lines to stop the URL being too long to read at a glance.

**Comment by jk** [comment]
:+1:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/a0a1af93af94f7862b5b1f8bbbd90e5f0ae5ecd3. Thank you for your contribution to Homebrew @cybertk; we appreciate it! :smiley_cat:

**Comment by cel4** [comment]
@cybertk are you still actively maintaining this formula? v1.0.4 is out. Unfortunately, simply updating the tag does not work for me.

**Comment by mikemcquaid** [comment]
> @cybertk are you still actively maintaining this formula? v1.0.4 is out. Unfortunately, simply updating the tag does not work for me.

@cel4 Could you try and open a pull request? This should help and I'm happy to walk you through anything else: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged

Thanks!

**Comment by cel4** [comment]
@mikemcquaid, thanks for the feedback - A pull request from my side does not make much sense at the moment. I am experiencing problems in the build phase and my knowledge of the go build system is very limited. At the moment I do not have the time to familiarize myself with the particularities of go's dependency management.

**Comment by cybertk** [comment]
See #49562

**Pull request closed without merging**